packages/battery/android/src/main/java/io/flutter/plugins/battery/BatteryPlugin.java

```diff
@@ -30,16 +30,16 @@ public static void registerWith(PluginRegistry.Registrar registrar) {
         new MethodChannel(registrar.messenger(), "plugins.flutter.io/battery");
     final EventChannel eventChannel =
         new EventChannel(registrar.messenger(), "plugins.flutter.io/charging");
-    final BatteryPlugin instance = new BatteryPlugin(registrar.context());
+    final BatteryPlugin instance = new BatteryPlugin(registrar);
     eventChannel.setStreamHandler(instance);
     methodChannel.setMethodCallHandler(instance);
   }
 
-  BatteryPlugin(Context context) {
-    this.context = context;
+  BatteryPlugin(PluginRegistry.Registrar registrar) {
+    this.mRegistrar = registrar;
   }
 
-  private final Context context;
+  private final PluginRegistry.Registrar mRegistrar;
```

Comment:
The `mMember` naming convention is not used in the existing code. I think we should either keep the current coding style, or apply the `m` prefix to all members.

Here and elsewhere.

Reply:
Changed to match existing style in each file.